kotlinx.fuzz.gradle/src/main/kotlin/kotlinx/fuzz/gradle/KFuzzPlugin.kt

```diff
@@ -12,6 +13,8 @@ import org.gradle.kotlin.dsl.*
 
 @Suppress("unused")
 abstract class KFuzzPlugin : Plugin<Project> {
+    val log = KLoggerFactory.getLogger(KFuzzPlugin::class)
```

Comment:
Ok, this time it's `LazyLogger` which always loads `NoOpLogger`.

Adding 
```
System.setProperty(
    KLoggerFactory.LOGGER_IMPLEMENTATION_PROPERTY, 
    GradleLogger::class.qualifiedName!!
)
```

to `Test.configureLogging` fixes it